pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -66,6 +66,14 @@ type StorageDeviceSet struct {
 	Placement       rookalpha.Placement          `json:"placement,omitempty"`
 	Config          StorageDeviceSetConfig       `json:"config,omitempty"`
 	DataPVCTemplate corev1.PersistentVolumeClaim `json:"dataPVCTemplate"`
+	// Following Fields will be present when we will get the BLOB from the UI
+	UserID         string `json:"userID"`
+	UserKey        string `json:"userKey"`
+	FilesystemName string `json:"filesystem-name,omitempty"`
+	PoolName       string `json:"pool-name,omitempty"`
+	ClusterName    string `json:"cluster-name,omitempty"`
+	MaxMonID       string `json:"maxMonId,omitempty"`
+	Fsid           string `json:"fsid,omitempty"`
```

Comment:
These were the fields that were coming from the JSON BLOB @aruniiird , looks good?. Thanks